### PR TITLE
feat: add oracle migration for v3.2

### PR DESCRIPTION
**Pull request by robert-zaremba**
## Description

Migration fixing tthe oracle params set.

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: app/upgrades.go [hunk 1]
```diff
@@ -23,31 +23,43 @@ func (app UmeeApp) RegisterUpgradeHandlers(experimental bool) {
 
 	app.registerV3_0Upgrade(upgradeInfo)
 	app.registerV3_1Upgrade(upgradeInfo)
+	app.registerV3_2Upgrade(upgradeInfo)
+}
+
+// performs upgrade from v3.1 -> v3.3
```
Comment (gsk967) [inline app/upgrades.go:29]: change v3.1 -> v3.2

**Review by gsk967 — APPROVED** [review]
LGTM

**Review by toteki — APPROVED** [review]

File: app/upgrades.go (hunk 1 above)
Comment (toteki) [inline app/upgrades.go:29]:
```suggestion
// performs upgrade from v3.1 -> v3.2
```

File: x/oracle/keeper/migrations.go [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+// NewMigrator creates a Migrator.
+func NewMigrator(keeper *Keeper) Migrator {
+	return Migrator{keeper: keeper}
+}
+
+// Migrate1to2 migrates from version 1 to 2.
+func (m Migrator) Migrate1to2(ctx sdk.Context) error {
+	m.keeper.SetStampPeriod(ctx, 1)
+	m.keeper.SetPrunePeriod(ctx, 1)
+	m.keeper.SetMedianPeriod(ctx, 1)
```
Comment (adamewozniak) [inline x/oracle/keeper/migrations.go:21]: Hmm feels weird to have these set to 1, but that's a better default than 0 since we'd have to remove validation

**Review by adamewozniak — APPROVED** [review]
looks good! & I agree with @RafilxTenfen , maybe we do a followup PR that enables the upgrade CI?

File: x/oracle/keeper/migrations.go (hunk 2 above)
Comment (robert-zaremba) [reply x/oracle/keeper/migrations.go:21]: yes, we just need to put some number there anyway.

**Comment by robert-zaremba** [comment]
>  maybe we do a followup PR that enables the upgrade CI?

We can, but we are testing it locally anyway. Other PRs will suffer.

**Comment by robert-zaremba** [comment]
so, maybe we can add the CI test run to the last PR?

**Pull request merged**